_dashboards/visualize/viz-index.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: default
+title: Visualize
+nav_order: 4
+has_children: true
+---
+
+# Visualize
+
+Data visualization is representing your data in a visual context. By visualizing your data, you translate complex, high-volume, or numerical data into a visual representation that is easier to process.
+
+OpenSearch Dashboards gives you data visualization tools to improve and automate the visual communication process. Using visual elements like charts, graphs, or maps to represent data gives you the tools to advance business intelligence and support data-driven decision-making and strategic planning.
+
+# Visualization types in OpenSearch Dashboards
+
+Dashboards has several visualization types to support your data analysis needs. The following sections give you an overview of the visualization types in Dashboards and their common use cases.
```

Comment:
nit, but do you also want to mention the visualization type names within each section? 

In the UI they can select these types: Area, Vertical bar, Horizontal bar, etc. . . 
or maybe this is already covered in the how-to section "Create visualization"

Reply:
We are grouping the types as area, line, and bar for this intro. We'll have subpages for each specific type, so we'll go into detail on the subpages.